classes/PodsUI.php

```diff
@@ -4653,9 +4655,13 @@ public function pagination( $header = false ) {
 
 		if ( false !== $this->pagination ) {
 			if ( 1 < $total_pages ) {
+				$first_link = esc_url( $request_uri . ( $append ? '&' : '?' ) . 'pg' . $this->num . '=1' );
```

Comment:
:no_entry_sign: Inline IF statements are not allowed

<sub>hash: 6da1b1864943789bd549a5e4363dd6a8</sub>